src/agent0/core/hyperdrive/interactive/hyperdrive_agent.py

```diff
@@ -153,6 +153,9 @@ def __init__(
         self.current_nonce = 0
 
     def _get_nonce_safe(self) -> Nonce:
```

Comment:
might as well make these top-line comments a function docstring, right?

```
Get agent nonces in a thread-safe manner.

We pass in the callable function to underlying ethpy calls so that we get nonce when we sign the transaction.

Returns
--------
Nonce
   The agent's current nonce value.
   ```